.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Task 🔧
```

Comment:
I would remove issues templates at all since all issues are supposed to be created in Che repo